build.gradle

```diff
@@ -119,31 +119,31 @@ allprojects {
                     }
                     maven {
 
-                        url "${artifactory_contextUrl}/libs-release-no-proxy"
+                        url "${artifactory_contextUrl}/libs-release"
 
                         if (project.hasProperty('artifactory_user') && project.hasProperty('artifactory_password'))
                         {
                             credentials {
                                 username = artifactory_user
                                 password = artifactory_password
                             }
-                            authentication {
-                                basic(BasicAuthentication) // enable preemptive authentication to get around https://www.jfrog.com/jira/browse/RTFACT-4434
-                            }
+                        }
+                        authentication {
+                            basic(BasicAuthentication)
```

Comment:
Community builds/tests are failing on TeamCity because they run without credentials.

Reply:
Oops. I put that back in the wrong place... Fixed now.